src/CalcManager/CalculatorHistory.cpp

```diff
@@ -53,7 +52,7 @@ unsigned int CalculatorHistory::AddToHistory(
     // in the history doesn't get broken for RTL languages
     spHistoryItem->historyItemVector.expression = L'\u202d' + generatedExpression + L'\u202c';
     spHistoryItem->historyItemVector.result = wstring(result);
-    addedIndex = AddItem(spHistoryItem);
+    unsigned int addedIndex = AddItem(spHistoryItem);
```

Comment:
Thank you for this one! We can even remove `addedIndex` and directly return addIndex